benchmarks/configurations.py

```diff
@@ -47,7 +47,7 @@
             "parameters": {
                 "height": 100,
                 "width": 100,
-                "homophily": 8,
+                "homophily": 0.4,
```

Comment:
this was 8 (the max for moore neighborhood, so it should be 1 now.

Reply:
@quaquel I will change it.